estatioapp/app/src/main/java/org/estatio/module/lease/dom/breaks/Lease_allBreakOptions.java

```diff
@@ -0,0 +1,17 @@
+package org.estatio.module.lease.dom.breaks;
+
+import org.apache.isis.applib.annotation.Mixin;
+
+import javax.inject.Inject;
+import java.util.List;
+
+@Mixin(method = "coll")
+public class Lease_allBreakOptions {
+
+    public List<BreakOption> coll() {
+        return ccoBreakOptionContributions.allBreakOptions();
+    }
+
+    @Inject
+    LeaseService ccoBreakOptionContributions;
```

Comment:
rename field to leaseService

Reply:
Removed this file all together, since the allBreakOptions method does not have any arguments, I couldn't get it to work as a mixin?

Reply:
You are right, you did the right thing.  That allBreakOptions wasn't contributing to anything.  My guess is that it's a left over from some earlier implementation, perhaps if the domain service had originally been nature=VIEW_MENU, which would also imply CONTRIBUTIONS as well as menu items, eg for prototyping.

Anyway, whatever, correct to just remove the file.